FWCore/Services/plugins/CondorStatusUpdater.cc

```diff
@@ -338,6 +339,24 @@ CondorStatusService::updateImpl(time_t sinceLastUpdate)
     }
 
     // Update storage account information
+    auto &stats = StorageAccount::summary();
+    updateStorageImpl(stats);
+    {
+      // Given there is some cost to reporting empty numbers, only push forward the secondary source info
+      // if one file was actually opened via such a source.
+      auto label = StorageAccount::setLabel("secondarysource");
```

Comment:
Better to create constants somewhere with this same since you are requiring multiple places to use the same labels. Also true for the 'tokenForStorageClassName' call.